src/versions.jl

```diff
@@ -223,7 +223,7 @@ function Base.intersect(A::VersionSpec, B::VersionSpec)
         i += 1
     end
     VersionSpec(ranges)
-end
+endi
```

Comment:
🤔

Reply:
GitHub web editor + iPad on-screen keyboard.

Reply:
Actually it’s my new syntax I’m proposing for Julia 2.0. Deprecate `end` in favor of `endi`.